README.md

```diff
@@ -109,3 +102,11 @@ That script will clone Theia from master branch and all Che related extensions f
 ## License
 
 - [Eclipse Public License 2.0](LICENSE)
+
+## Join the community
+
+The Eclipse Che community is globally reachable through public chat rooms, mailing list and weekly calls.
+See https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che.html#joining-the-community
+
+Che-Theia is mainly maintained by the Che IDE2 team a.k.a Selene.
```

Comment:
hello, why not ask directly on eclipse-che channel ?

Reply:
for global questions -> https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che.html#joining-the-community provides links to eclipse-che channel and mailing list.

For che-theia related questions ide2 channel is more suited because we have the right people watching that channel and expecting to have questions related to che-theia only.